js/samples/06.assistants.a.mathBot/README.md

```diff
@@ -0,0 +1,179 @@
+# Math Tutor Assistant
+
+This example shows how to create a basic conversational experience using OpenAI's Assistants API's. It leverages OpenAI's Code Interpreter tool to create an assistant that's an expert on math.
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
```

Comment:
Please update sample formats to match current -- echobot on `main` should be a good example. Some things that are different:

1. Some of the directions were moved to global docs
2. Every sample should have a note of what version of yarn is needed
3. More focus is put on covering what the app does and how to interact with it to see the features.

Reply:
I was waiting to get the readme from the TTK template. Theirs is good.